deploy/aws/deploy/app/api/templates/deployment.yaml

```
@@ -19,6 +19,9 @@ spec:
       - name: {{ .Values.service }}
         image: {{ .Values.awsAccountId }}.dkr.ecr.{{ .Values.awsRegion }}.amazonaws.com/{{ .Values.service }}:{{ $.Values.imagesVersion }}
         imagePullPolicy: Always
+        envFrom:
+          - secretRef:
+              name: secret_name
```

hmm, let's create a secret ref name dynamically:

Pass in helm upgrade command (`deploy/aws/deploy/script/src/index.js` line 56) current environment (staging/production) 

And update secret ref name using variables:
```suggestion
              name: api-{{ .Values.environment }}-secret
```

also, we need to configure ConfigMaps using (differences here: https://k21academy.com/docker-kubernetes/configmaps-secrets/#:~:text=What%20is%20the%20difference%20between,API%20keys%2C%20or%20TLS%20certificates.)